src/main/java/duke/command/InputCommand.java

```diff
@@ -0,0 +1,14 @@
+package duke.command;
+
+public enum InputCommand {
+    TODO,
+    DEADLINE,
+    EVENT,
+    LIST,
+    DONE,
+    DELETE,
+    INVALID,
+    BYE,
+    GET_TASK,
+    FIND,
```

Comment:
May I ask what is the difference between both of these commands? Maybe you want to rename it something more intuitive (SHOW, FIND_KEYWORD, FIND_DATE, etc) ?